aep/general/0154/aep.md.j2

```diff
@@ -23,20 +23,28 @@ ETag: "55cc0347-66fc-46c3-a26f-98a9a7d61d0e"
 ```
 
 The ETag **must** be provided by the server on output, and values **should**
-conform to [RFC 7232][]. Resources **must** support the `If-Match` header (and
+conform to [RFC 9110][]. Resources **must** support the `If-Match` header (and
 **may** support the `If-None-Match` header) if and only if resources provide
 the ETag.
 
-**Note:** ETag values **must** include quotes as described in [RFC 7232][]. For
+**Note:** ETag values **must** include quotes as described in [RFC 9110][]. For
 example, a valid ETag is `"foo"`, not `foo`.
 
 ETags **must** be based on an opaque checksum or hash of the resource that
 guarantees it will change if the resource changes.
 
 ### Condition headers
 
-Services that provide ETags **should** support the `If-Match` and
-`If-None-Match` headers on incoming requests:
+Services that provide ETags **should** support all applicable
+[preconditions](https://www.rfc-editor.org/rfc/rfc9110.html#name-preconditions)
+documented in RFC 9110, including:
+
+- `If-match`
+- `If-None-Match`
+- `If-Modified-Since`
+- `If-Unmodified-Since`
```

Comment:
It's a little odd that support for `If-Modified-Since` and `If-Unmodified-Since` should be conditioned on whether Etags are provided, as these headers use date values. Not sure if a change is needed here, but just wanted to mention this.

Reply:
oops true, good point. Let me rewrite this and omit those. I'll try to add a new section that might just provide general guidance to adhere to RFC 9110.